backend/integration/webpack.config.js

```diff
@@ -29,23 +32,14 @@ module.exports = {
   },
 
   module: {
-
-    // require
-    unknownContextRegExp: /$^/,
-    unknownContextCritical: false,
-
-    // require(expr)
-    exprContextRegExp: /$^/,
-    exprContextCritical: false,
-
-    // require("prefix" + expr + "surfix")
-    wrappedContextRegExp: /$^/,
-    wrappedContextCritical: false,
-
-    loaders: [{
-      test: /\.js$/,
-      loader:  'babel',
-      exclude: /node_modules/,
-    }],
+    rules: [
+      {
+        test: /\.js$/,
+        loader: 'babel-loader',
+        options: {
+          ...JSON.parse(readFileSync(resolve(__dirname, '../../.babelrc'))),
```

Comment:
Is this spread operator really needed? You could remove the object braces and set `options` to the result of JSON.parse

Maybe I'm missing something

Reply:
Originally, I thought I might add some custom options to individual configs but that turned out not to be necessary.